CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -342,6 +342,26 @@ protected function getTemplateUpdates() {
           ['name' => 'payment_or_refund_notification', 'type' => 'text'],
         ],
       ],
+      [
+        'version' => '5.53.alpha1',
+        'upgrade_descriptor' => ts('Replace date_format modifier with crmDate modifier in message templates for php8.1 support'),
+        'templates' => [
```

Comment:
The last time I added a section like this for a change people didn't need to do yet, the consensus was that it's not worth scaring people into thinking they need to go through the process of reapplying their customizations. So maybe this PR should be for new installs only right now?

Reply:
ok have removed this now